src/main/kotlin/com/deviceinsight/helm/ResolveHelmMojo.kt

```diff
@@ -60,85 +57,77 @@ open class ResolveHelmMojo : AbstractHelmMojo() {
 	@Component
 	private lateinit var repositorySystem: RepositorySystem
 
-	protected lateinit var helm : String
+	protected lateinit var helm: String
 
-	protected fun resolveHelmBinary(): String {
+	override fun execute() {
+		checkHelmVersion()
+		helm = resolveHelmBinary()
+	}
 
-		val platformIdentifier = PlatformDetector.detectHelmReleasePlatformIdentifier()
-		val helmArtifact: Artifact =
-			repositorySystem.createArtifactWithClassifier(helmGroupId, helmArtifactId, helmVersion, "binary",
-				platformIdentifier)
+	private fun checkHelmVersion() {
+		val (majorVersion, minorVersion) = helmVersion.split('.').map(String::toInt)
```

Comment:
Maybe add a sanity check (i.e. check that the format actually matches `\d+(\.\d+){2}`)?